### DISCUSSION TITLE
lc/319/

### DISCUSSION THREAD

**Discussion by giscus** [doocs/leetcode#2754] [Announcements]
# lc/319/

多种编程语言实现 LeetCode、《剑指 Offer（第 2 版）》、《程序员面试金典（第 6 版）》题解

https://leetcode.doocs.org/lc/319/

**Comment by lazyJLBL** [comment]
emmm，这个有解释吗，有点看不懂哎

**Reply by yanglbme** [reply]
题解已更新

**Reply by lazyJLBL** [reply]
欧克欧克，感谢y神

**Comment by zhengmingliang** [comment]
这解法一点都不程序员，转换为了数学找规律问题了，不过也变相说明要提高效率，就要把复杂问题转换为数学问题

**Reply by lazyJLBL** [reply]
但是这是算法题呀，算法题必须得在一定的时间和空间内得到答案
